web/html/src/manager/recurring/recurring-actions-list.tsx

```diff
@@ -84,25 +132,32 @@ class RecurringStatesList extends React.Component<Props, State> {
           </div>
           <div>
             <Table
-              data={this.props.data}
+              selectable={false}
+              data={isFilteredList ? this.state.schedules : "/rhn/manager/api/recurringactions"}
               identifier={(action) => action.recurringActionId}
               /* Using 0 to hide table header/footer */
-              initialItemsPerPage={this.props.disableCreate ? pageSize : 0}
-              emptyText={t(
-                "No schedules created." + (this.props.disableCreate ? "" : " Use Create to add a schedule.")
-              )}
+              initialItemsPerPage={disableCreate ? pageSize : 0}
+              emptyText={t("No schedules created." + (disableCreate ? "" : " Use Create to add a schedule."))}
```

Comment:
I'm not sure if this works correctly with the translation tooling? Would be nice to check just in case.